src/commonMain/kotlin/example.kt

```diff
@@ -17,8 +17,12 @@ typealias NastyLettuce = CookingException.LettuceIsRotten
 typealias KnifeIsDull = CookingException.KnifeNeedsSharpening
 typealias InsufficientAmountOfLettuce = CookingException.InsufficientAmount
 
-fun takeFoodFromRefrigerator(): Either<NastyLettuce, Lettuce> = Right(Lettuce)
-fun getKnife(): Either<KnifeIsDull, Knife> = Right(Knife)
+fun takeFoodFromRefrigerator(): Either<NastyLettuce, Lettuce> =
+  Right(Lettuce)
+
+fun getKnife(): Either<KnifeIsDull, Knife> =
```

Comment:
I'd use suspend functions in the example so it's clear it's now supported in `either` blocks